cmd/ctr/commands/run/run_unix.go

```diff
@@ -350,8 +349,12 @@ func NewContainer(ctx context.Context, client *containerd.Client, cliContext *cl
 				Path: nsPath,
 			}))
 		}
+		var cdiDeviceIDs []string
 		if cliContext.IsSet("gpus") {
-			opts = append(opts, nvidia.WithGPUs(nvidia.WithDevices(cliContext.IntSlice("gpus")...), nvidia.WithAllCapabilities))
```

Comment:
@mxpv would you prefer that this be opt-out? (I wanted to ask that before removing the code that this calls).

Reply:
I don't think we have any backward compatibility promise for `contrib/` and we no longer use it in the codebase, so it should be fine to remove it I think.